examples/update_request.json

```diff
@@ -36,7 +36,7 @@
   "structural": {
     "hasAgreement": "druid:dd327qr3670",
     "hasMember": [],
-    "isTargetOf": "",
+    "isTargetOf": null,
```

Comment:
reminder to self: document this in metadata guidelines per original discussion + agreement on this practice with jcoyne